GetStream/stream-chat-react-native
# fix(SampleApp): update references to renamed functions from image gallery context

## 🎯 Goal

In the current version of the SampleApp, the app crashes if you select "Photos & Videos" for a chat.

This is because we've updated the naming of the values exposed by `ImageGalleryContext`.

This PR fixes that.

See [this Slack thread](https://getstream.slack.com/archives/C016MJGGSF9/p1666803497919339)

## 🛠 Implementation details

I've kept the old names for use locally in the SampleApp (see code), since the updated names exposed clash with other variables in the relevant hook

## 🎨 UI Changes

none

## 🧪 Testing

1. Open the SampleApp
2. Open a channel
3. Click the channel icon in the top right corner
4. Click "Photos & Videos"
5. See that the app doesn't crash

## ☑️ Checklist
